source/layouts/base.erb

```diff
@@ -19,9 +20,7 @@
     <link rel="shortcut icon" href="https://assets.lrug.org/favicon.ico" />
     <link rel="icon" href="https://assets.lrug.org/favicon.ico" />
 
-    <%= stylesheet_link_tag "normalize", "all" %>
-    <%= javascript_include_tag  "all" %>
-
+    <%= stylesheet_link_tag "site" %>
```

Comment:
Awesome I can switch that back :) I think I just renamed it to make breaking out the new styles side by side & didn't know how readme.lrug.org pulled styled in.

That said! In the future I should probably aim to spend some time updating the readme to be independent - It could limit our options if we do want to embrace cache breakers & webpack.